src/HashBus.Projector.MostMentioned/MostMentionedProjection.cs

```diff
@@ -20,17 +21,17 @@ public MostMentionedProjection(IRepository<string, IEnumerable<Mention>> mention
             this.mentions = mentions;
         }
 
-        public void Handle(TweetAnalyzed message)
+        public Task Handle(TweetAnalyzed message, IMessageHandlerContext context)
         {
             if (!message.Tweet.UserMentions.Any())
             {
-                return;

```

Comment:
given that we're targeting `4.6.1` we can use `Task.CompletedTask` as return value